### PR TITLE
Add device configuration option to volume widget

### PR DESCRIPTION
**Pull request by bhartshorn**
Hi. This is in reference to my issue #41. 

I'm a total noob at Lua, but I think I ironed out any bugs I introduced. Specifically I was confused on using _local_, so I'm not sure if I came up with a good solution. Point me the right way and I will fix it. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -447,8 +447,10 @@ Provides volume levels and state of requested mixers.
 Supported platforms: Linux (required tool: amixer), FreeBSD.
 
 - Arguments (per platform):
-  * Linux: takes the ALSA mixer control as an argument, i.e. `"Master"`,
-    optionally append the card ID or other options, i.e. `"PCM -c 0"`
+  * Linux: takes either a single argument containing the ALSA mixer control as
+    an argument, i.e. `"Master"`, or a table, with the first argument being the
+    ALSA mixer control and the second being the ALSA device ID, i.e.
+    `{"Master", "pulse"}`.
```
Comment (Mic92) [inline README.md]: Btw. we have also a dedicated pulseaudio widget:
https://github.com/Mic92/vicious/blob/master/contrib/pulse_all.lua

File: README.md [hunk 2]
```diff
@@ -447,8 +447,10 @@ Provides volume levels and state of requested mixers.
 Supported platforms: Linux (required tool: amixer), FreeBSD.
 
 - Arguments (per platform):
-  * Linux: takes the ALSA mixer control as an argument, i.e. `"Master"`,
-    optionally append the card ID or other options, i.e. `"PCM -c 0"`
+  * Linux: takes either a single argument containing the ALSA mixer control as
+    an argument, i.e. `"Master"`, or a table passed as command line arguments
+    to [amixer(1)](https://linux.die.net/man/1/amixer),
+    i.e `{"PCM", "-c", "0"}` or `{"Master", "-D", "pulse"}`.
```
Comment (Mic92) [inline README.md]: How about the following: this makes it more flexible.

File: widgets/volume_linux.lua [hunk 3]
```diff
@@ -26,8 +26,22 @@ local function worker(format, warg)
         ["off"] = "♩"  -- "M"
     }
 
+    local mixer = nil
+    local device = nil
```
Comment (Mic92) [inline widgets/volume_linux.lua]: using local is always a good idea. The default value of a variable should be nil already.

File: README.md (hunk 2 above)
Comment (bhartshorn) [reply README.md]: Looks good and the command seems to work fine with the quotes on my end. I like this method better. Thanks.

**Pull request merged**